datalad/crawler/nodes/misc.py

```diff
@@ -56,14 +57,19 @@ def get_values(self, *keys):
         return [[d[k] for k in keys] for d in self.data]
 
     def __call__(self, data):
-        # ??? for some reason didn't work when I made entire thing a list
         if self.keys:
-            raise NotImplementedError("Jason will do it")
+            for k in data.keys():
+                size = len(self.keys)
+                for i in range(0, size):
+                    key = self.keys[i]
+                    if key == k:
+                        self.data.append({key: data.get(key)})
```

Comment:
Let's work on this piece... 
Python is great! You could do just
`for key in self.keys:`
To traverse the list of keys. Adjust the code with that in mind. Then note my note I will leave in the unittest